packages/tooltip/src/styledComponents/Tooltip.js

```diff
@@ -13,7 +13,7 @@ const StyledTooltip = styled.div`
   box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.16);
   border: solid 1px rgba(0, 0, 0, 0.16);
   border-radius: 4px;
-  white-space: pre;
```

Comment:
Why was this removed?

I don't know if it's necessary, but let's check. @fczuardi  do you remember why was this rule added?

Reply:
This was preventing a long text from line breaking, but I think I can leave it on if this effect is expected.

Reply:
With `white-space: pre;`
![captura de tela de 2018-03-14 09-11-08](https://user-images.githubusercontent.com/17071199/37401701-b2c1f86a-2767-11e8-9a3e-8bec5d469808.png)

Without:
![captura de tela de 2018-03-14 09-10-44](https://user-images.githubusercontent.com/17071199/37401720-bdacfafe-2767-11e8-8d3a-379a4d1189d9.png)

Reply:
We talked about this, but to keep documented:
Let's keep the `white-space: pre;`